stories/Molecules/DownloadRow/DownloadRow.jsx

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import './downloadrow.scss';
 import { Checkbox } from '../../Components/Forms/Checkbox/Checkbox';
+import { Heading } from '../../Atom/Typography/Heading/Heading';
 
 export const DownloadRow = ({
   title, subtitle, value, datalang, ...args
 }) => (
   <div className="selectbox-container" data-lang={datalang}>
     <div className="selectbox-container__data">
-      <div className="selectbox-container__data-headertext">{title}</div>
-      <div className="selectbox-container__data-subtext">{subtitle}</div>
+      <Heading type="6" label={title} />
+      <Heading className="selectbox-container__data subtext" type="6" label={subtitle} />
```

Comment:
@pallavikadam-srijan We don't need H6 for both elements. It will only apply on title. Also we do not need classes on both. You can use the following structure and style:
```
.selectbox-container__data { 
  h6 { ... }
  span { ... }
}
```
I do also see spacing discrepancies between title and grey text. 
Kindly validate with Figma for margin/spacing between the two text.